File: Conexion base de datos/JAVA/mongodb.java
import com.mongodb.client.MongoClients;
import com.mongodb.client.MongoClient;

public class MongoDBConnection {
    public static void main(String[] args) {
        String connectionString = "mongodb://localhost:27017";
        
        try (MongoClient mongoClient = MongoClients.create(connectionString)) {
            System.out.println("✅ Conexión exitosa a MongoDB");
            
            // Verificación adicional (opcional)
            System.out.println("Versión del servidor: " + 
                mongoClient.getDatabase("admin")
                          .runCommand(new Document("serverStatus", 1))
                          .getString("version"));
        } catch (Exception e) {
            System.err.println("❌ Error de conexión: " + e.getMessage());
        }
    }
}

File: Conexion base de datos/JAVASCRIPT/mongodb.js
const { MongoClient } = require('mongodb');

async function connectMongoDB() {
  const uri = "mongodb://localhost:27017";
  const client = new MongoClient(uri);

  try {
    await client.connect();
    console.log("✅ Conexión exitosa a MongoDB");
    
    // Verificación adicional (opcional)
    const adminDb = client.db('admin');
    const serverStatus = await adminDb.command({ serverStatus: 1 });
    console.log(`Versión de MongoDB: ${serverStatus.version}`);
    
  } catch (err) {
    console.error("❌ Error de conexión:", err.message);
  } finally {
    await client.close();
  }
}

connectMongoDB();